MdePkg/MdePkg.ci.yaml

```diff
@@ -117,7 +117,12 @@
 
     ## options defined ci/Plugin/DscCompleteCheck
     "DscCompleteCheck": {
-        "IgnoreInf": [""],
+        "IgnoreInf": [
+            # MU_CHANGE Ignore StackCookie libs for toolchains that don't support it.
+            "MdePkg/Library/BaseBinSecurityLibNull/BaseBinSecurityLibNull.inf",
+            "MdePkg/Library/BaseBinSecurityLibRng/BaseBinSecurityLibRng.inf",

```

Comment:
Drop the BaseBinSecurityLib stuff, not for this PR and not existent any longer

Reply:
dropped, thanks!